dev/com.ibm.ws.wsat.webclient/bnd.bnd

```diff
@@ -23,16 +33,15 @@ Export-Package: \
  com.ibm.ws.wsat.webservice.client.wsat
  
 Include-Resource: \
-  @./resources!/META-INF/**
+  META-INF=../com.ibm.ws.wsat.webclient/resources/META-INF
```

Comment:
This is wrong of course.  You should be able to do what you had before or `META-INF=resources/META-INF`